tribuo-classification/host/build.gradle

```diff
@@ -20,17 +20,4 @@ dependencies {
 shadowJar {
     archiveAppendix.set(mode)
     archiveClassifier.set("")
-}
```

Comment:
Let's add a newline at the end of this file. But why are we removing the code below?